change-notes/1.21/analysis-csharp.md

```diff
@@ -4,7 +4,7 @@
 
 | **Query**                    | **Expected impact**    | **Change**                        |
 |------------------------------|------------------------|-----------------------------------|
-
+| Constant condition (`cs/constant-condition`) | Fewer results | Results have been removed where the `null` value is in a conditional expression in the left hand side of a null-coalescing expression. For example, in `(a ? b : null) ?? c`, `null` is not considered to be a constant condition. |
```

Comment:
Results have been removed where the `null` value is in a conditional expression ~in~ _on_ the left hand side of a null-coalescing expression?

And is it fair to say that this will result in `Fewer false positives` rather than simply `Fewer results`?